lib/govuk_publishing_components/presenters/intervention_helper.rb

```diff
@@ -1,11 +1,13 @@
 module GovukPublishingComponents
   module Presenters
     class InterventionHelper
-      def initialize(local_assigns)
+      def initialize(options = {})
         @accessible_text_suffix = I18n.t("components.intervention.accessible_link_text_suffix")
-        @query_string = local_assigns[:query_string]
-        @suggestion_link_text = local_assigns[:suggestion_link_text]
-        @suggestion_link_url = local_assigns[:suggestion_link_url]
+        @params = options[:params] || nil
+        @query_string = options[:query_string] || nil
```

Comment:
Is defaulting these values to `nil` necessary? I thought that would be the default already if not supplied.